app/src/androidTest/java/org/schabi/newpipe/testUtil/TrampolineSchedulerRule.kt

```diff
@@ -0,0 +1,34 @@
+package org.schabi.newpipe.testUtil
+
+import io.reactivex.rxjava3.android.plugins.RxAndroidPlugins
+import io.reactivex.rxjava3.plugins.RxJavaPlugins
+import io.reactivex.rxjava3.schedulers.Schedulers
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+/**
+ * Always run on [Schedulers.trampoline]
+ */
+class TrampolineSchedulerRule : TestRule {
+
+    private val scheduler = Schedulers.trampoline()
```

Comment:
After reading some resources on the internet I understand why you used `trampoline()`, but maybe add a comment to make it clearer

Reply:
done